Calculator/Calculator/Program.cs

```diff
@@ -0,0 +1,89 @@
+using System;
+
+namespace Calculator
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int previous;
+            int current;
+            string operation;
+
+            while(true)
+            {
+                StartCalculation:
+                while(true)
+                {
+                    Console.Write("Введите число: ");
+                    var priv = Console.ReadLine();
+                    try
+                    {
+                        previous = int.Parse(priv);
+                        break;
+                    }
+                    catch (Exception)
+                    {
+                        Console.WriteLine("Это не число");
+                    }
+                }
+
+                while(true)
+                {
+                    Console.Write("Введите операцию: ");
+                    operation = Console.ReadLine();
+                    if (operation == "!")
+                    {
+                        Console.WriteLine($"Факториал числа {previous} равен {Factorial(previous)}");
+                        goto StartCalculation;
```

Comment:
Please don't use this, you just adding mess in the code

Reply:
Fixed in commit 90e9d44792f4e9d179092415f8fda8d02c19f137